test/create-open.test.js

```diff
@@ -207,6 +207,36 @@ Object.keys(testAPIs).forEach(API => {
             assert.deepEqual(db.access.write, [orbitdb.identity.id])
           })
         })
+        describe('Meta', function() {
+          before(async () => {
+            if (db) {
+              await db.close()
+              await db.drop()
+            }
+          })
+
+          afterEach(async () => {
+            if (db) {
+              await db.close()
+              await db.drop()
+            }
+          })
+
+          it('creates a manifest with no meta field', async () => {
+            db = await orbitdb.create('no-meta', 'feed')
+            const manifest = await io.read(ipfs, db.address.root)
+            assert.strictEqual(db.options.meta, undefined)
+            assert.deepStrictEqual(Object.keys(manifest).filter(k => k === 'meta'), [])
```

Comment:
Could we use `assert.strictEqual(manifest.meta, undefined)` here to match the convention in the rest of the codebase? What do you think?

Reply:
yes that's a good idea

Reply:
i guess the like 228 and 229 are also redundant?

Reply:
Hmm could be but I don't see them as redundant since you're checking whether `meta` is set in the db options, while 229 checks that `meta` has been written to ipfs.

Reply:
made a commit to try and solve the code convention issue. it makes lines 228 and 229 sort of redundant so we could remove line 228, but personally it doesnt seem like a big deal either way and also keeps both tests more symmetrical. lmk what you think